testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1542,7 +1542,7 @@ QUnit.module('options', {
             autoValue = autoValue();
         }
 
-        assert.equal(initialValue, autoValue, 'initial value equal auto value');
+        assert.equal(autoValue, initialValue, 'initial value equal auto value');
```

Comment:
Does order matter?

Reply:
Unfortunately, it looks like a flasher

Reply:
Seems that old order more logically, but ok.